### DISCUSSION TITLE
Add APNG support

### DISCUSSION THREAD

**Discussion by Poker-sang** [SixLabors/ImageSharp#2510] [Ideas] [answered]
Support APNG decoding and encoding. This allows us to have other options than GIFs when making animated images.

**Comment by JimBobSquarePants** [comment, marked as answer]
APNG is an [unofficial](https://sourceforge.net/p/png-mng/mailman/message/131482/) extension to the PNG specification with very limited support. As such it will not be a focus for the library. We will however welcome external contribution adding support.

**Reply by Poker-sang** [reply]
> APNG is an [unofficial](https://sourceforge.net/p/png-mng/mailman/message/131482/) extension to the PNG specification with very limited support. As such it will not be a focus for the library.

No worries. I tried to implement one and would like to pr.
